### ISSUE TITLE
Utgarde Pinnacle: King Ymiron does not become hostile cannot be attacked

### ISSUE THREAD

**Issue by taichouVu** [dalaranwow/dalaran-wow#5956]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Enter room, ,
2. kill minions
3.  walk up to Ymiron, dance
![WoWScrnShot_052521_094022](https://user-images.githubusercontent.com/79895201/119536902-64b2e600-bd3e-11eb-8161-9b0f8d5dd33f.jpg)

He is supposed to agro/targetable

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this is the first time I've seen this - I will take a look.

**Comment by taichouVu** [comment]
Unfortunately the King is still untargetable. My new team just encountered the same thing. :(
![WoWScrnShot_090821_232036](https://user-images.githubusercontent.com/79895201/132634373-5a423b43-08a9-4b4d-aaf3-b469803a0475.jpg)
